tests/DBusTests/Util.hs

```diff
@@ -297,3 +309,9 @@ assertThrows check f = do
     case result of
         Left ex -> assertBool ("unexpected exception " ++ show ex) (check ex)
         Right _ -> assertFailure "expected exception not thrown"
+
+nonWindowsTestCase :: TestName -> Assertion -> TestTree
```

Comment:
Unix file descriptor passing should not work on Windows, so this is a hack to stop running the tests there.

I have not actually tested on anything other than Linux, though.